convex/subscriptions.ts

```
@@ -0,0 +1,679 @@
+      status: args.status,
+      start_date: args.startDate,
+      cancel_at: args.cancelAt,
+      canceled_at: args.canceledAt,
+      ended_at: args.endedAt,
+      plan_type: args.planType,
+      team_name: args.teamName,
+      quantity: args.quantity,
+      updated_at: Date.now(),
+    };
+
+    let newSubscription;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)